src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: $main-font;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  color: $color-secondary;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
>css properties order should be like that
http://codeguide.academy/html-css.html#css-order
fix all such cases

not fixed